assets/section-image-banner.css

```diff
@@ -49,6 +79,51 @@
   .banner--large:not(.banner--adapt) {
     min-height: 72rem;
   }
+
+  .banner__content.banner__content--top-left {
+    align-items: flex-start;
+    justify-content: flex-start;
+  }
+
+  .banner__content.banner__content--top-center {
+    align-items: flex-start;
+    justify-content: center;
+  }
+
+  .banner__content.banner__content--top-right {
+    align-items: flex-start;
+    justify-content: flex-end;
+  }
+
+  .banner__content.banner__content--middle-left {
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  .banner__content.banner__content--middle-center {
+    align-items: center;
+    justify-content: center;
+  }
+
+  .banner__content.banner__content--middle-right {
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  .banner__content.banner__content--bottom-left {
+    align-items: flex-end;
+    justify-content: flex-start;
+  }
+
+  .banner__content.banner__content--bottom-center {
+    align-items: flex-end;
+    justify-content: center;
+  }
+
+  .banner__content.banner__content--bottom-right {
+    align-items: flex-end;
+    justify-content: flex-end;
+  }
```

Comment:
I don't think you need this added specificity here. It looks like this is being overwritten because `.banner__content` has other assignments below this (line 174). 

Can we move this to line 190 and group it with this code? Then we can remove the extra `.banner__content`.

```
@media screen and (min-width: 750px) {
  .banner__content {
    padding: 5rem;
  }
}
```